src/services/auth/auth.middleware.spec.ts

```diff
@@ -34,6 +34,10 @@ describe('AuthMiddleware', () => {
     const key = new Ed25519Provider(seedBuf)
     const did = new DID({ provider: key, resolver: KeyResolver.getResolver() })
 
+    beforeAll(() => {
+        process.env.ENVIRONMENT = 'local'
```

Comment:
I see you've added this to the tests. Is this a bug fix for the tests? I don't see it reference elsewhere in this PR

Reply:
I don't think it actually fixed the pipelines; there was another change in that commit, which I think may also have done it. If you look at the commit, it was added to the other test files as well, mostly as a preventative measure to prevent it from trying to make hive calls and stuff while testing.

https://github.com/spknetwork/acela-core/pull/22/files#diff-506332b712d92aab189d6b80726efe16753731d21781bba882c469cf8439be59R41

https://github.com/spknetwork/acela-core/pull/22/files#diff-250c55def2a4c16853f447dd8842a6dd75c3e2240fd9ad426dee29b161eb048aR34

https://github.com/spknetwork/acela-core/pull/22/files#diff-e40e2b22df054150a4d1dd213f2fca7cad1890bf4e2d9aed030686fc2a2c5cc8R39

Reply:
this is actually unnecessary since all tests are ran in local mode anyway, will remove